guides/source/active_job_basics.md

```diff
@@ -99,7 +123,10 @@ That's it!
 Job Execution
 -------------
 
-If no adapter is set, the job is immediately executed.
+For enqueuing and executing jobs you need to set up a queuing backend, that is to
+say you need to decide for a 3rd-party queuing implementation that Rails should use.
```

Comment:
I don't think this is necessarily true, since the inline adapter will be used _if you don't choose_.